### PR TITLE
{devel}[GCCcore/8.3.0] SQLite v3.29.0

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

depends on ~#8619~ ~#8628~

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/6f60f06c28c17c72d7f4e5dcc8fdc105 for a full test report.

**Comment by vanzod** [comment]
Re-trigger Travis after merging deps

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/a9793895416501600f81aa0270056b84 for a full test report.

File: easybuild/easyconfigs/s/SQLite/SQLite-3.29.0-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+version_str = '%%(version_major)s%s00' % ''.join('%02d' % int(x) for x in version.split('.')[1:])
+source_urls = ['http://www.sqlite.org/2019/']
+sources = ['sqlite-autoconf-%s.tar.gz' % version_str]
+checksums = ['8e7c1e2950b5b04c5944a981cb31fffbf9d2ddda939d536838ebc854481afd5b']
+
+builddependencies = [('binutils', '2.32')]
```
Comment (verdurin) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.29.0-GCCcore-8.3.0.eb]: I've rather lost track of all the changing policies here, but assume this is okay.

Comment (Micket) [reply easybuild/easyconfigs/s/SQLite/SQLite-3.29.0-GCCcore-8.3.0.eb]: I've had others request that I put single item lists on one line before, so I went and did this for a whole bunch (of which SQLite was one of them). I don't really have a strong opinion myself.

Comment (verdurin) [reply easybuild/easyconfigs/s/SQLite/SQLite-3.29.0-GCCcore-8.3.0.eb]: Ah, didn't mean the syntax. It was more the use of `GCCcore` with the `builddependency` on `binutils`. Haven't been able to attend too many of the conference calls lately, so just wanted to check this was okay.

Comment (Micket) [reply easybuild/easyconfigs/s/SQLite/SQLite-3.29.0-GCCcore-8.3.0.eb]: I see. i targeted the suggested binutils for 2019b; #8567

**Comment by verdurin** [comment]
Going in, thanks @Micket!

**Pull request merged**